templates/etc/squid3/squid.conf.j2

```diff
@@ -2276,7 +2276,7 @@ http_port {{ squid_http_port }} transparent
 #			See ICP and HTCP options below for additional details.
 #
 #
-{% if squid_cache_peering %}
+{% if squid_cache_peering is defined %}
```

Comment:
Just with {% if squid_cache_peering %} the cache_peer want set, becasue as i get it just check as boolen.
And {% if squid_cache_peering is defined %} check also if there is item too.